app/controllers/application_controller.rb

```
@@ -9,7 +9,9 @@
 class ApplicationController < ActionController::API
   include Authorization
   include JwtToken
-  
+  prepend_before_action :set_response, only: %i[show update]
+  before_action :find_and_authorize_map_for_create, only: %i[create]  # changed from prepend_before_action
```

RuboCop (1.86.1)

[convention] Unnecessary spacing detected.

(Layout/ExtraSpacing)